commonware/response/middleware.py

```diff
@@ -43,14 +43,14 @@ class StrictTransportMiddleware(object):
     """
     Set the Strict-Transport-Security header on responses. Use the
     STS_MAX_AGE setting to control the max-age value. (Default: 1 year.)
-    Use the STS_SUBDOMAINS boolean to add includeSubdomains.
+    Use the SECURE_HSTS_INCLUDE_SUBDOMAINS boolean to add includeSubdomains.
```

Comment:
Yup. This should be removed and the docs updated to point to [Django's `SecurityMiddleware`](https://docs.djangoproject.com/en/1.8/ref/middleware/#security-middleware).